Update Connector Status Codes (#1528)
FusionAuth/fusionauth-site: site/docs/v1/tech/connectors/generic-connector.adoc

A unique name to identify the Connector. This name is for display purposes only and it can be modified later if desired.

[field]#Authentication URL# [required]#Required#::

> The fully qualified URL of the API endpoint. The connector will send an HTTP POST request to this URL to authenticate the user. The format returned will be identical to the link:/docs/v1/tech/apis/login[Login API]See the link:/docs/v1/tech/connectors/generic-connector#response[response section] for status codes accepted by FusionAuth from your connector.

Reviewer: ```suggestion
The fully qualified URL of the API endpoint. The connector will send an HTTP POST request to this URL to authenticate the user. The format returned will be identical to the link:/docs/v1/tech/apis/login[Login API]. See the link:/docs/v1/tech/connectors/generic-connector#response[response section] for status codes accepted by FusionAuth from your connector.
```

When you are not migrating a user,the [field]#user.id# must be the same across subsequent logins.
====

> If the user cannot be authenticated, then you should return a non-`200` status code.

Reviewer: ```suggestion
If the user cannot be authenticated, then you should return a `404`.
```

You can avoid issues by having your API endpoint write login data and completely close the connection as quickly as possible.
You can load test your Connector using [FusionAuth's load testing framework](/docs/v1/tech/admin-guide/monitor#load-testing) or any other HTTP based load testing framework.
Additionally, ensure the API endpoint is sized correctly and not starved of resources.

> If the user cannot be authenticated, you should return the appropriate status code and data as quickly as possible.

Reviewer: ```suggestion
You should return the appropriate status code and data as quickly as possible.
```
